app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,39 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: { class: "form-horizontal"} do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            .form-group
+              %label.control-label.col-sm-3
+              .col-sm-3
+                = f.text_field :title, class: "form-control"
+            .form-group
+              %label.control-label.col-sm-3
+              .col-sm-6
+                = f.select :category_id, options_for_select(Category.all.map {|category| [category.name, category.id]}), class: "form-control"
+            .form-group
+              %label.control-label.col-sm-3
+              .col-sm-6
+                = f.text_area :description, rows: 8, class: "form-control"
```

Comment:
Refactor all input fields and read bootstrap form docs. you don't need to have this much html code.